modules/ocf_mirrors/manifests/monitoring.pp

```diff
@@ -15,26 +16,26 @@
   if $type == 'debian' {
     $local_url = "${local_base}/dists/${dist_to_check}/Release"
     $upstream_url = "${upstream_base}/dists/${dist_to_check}/Release"
-  } elsif $type == 'unix_timestamp' or $type == 'datetime' {
+  } elsif $type == 'unix_timestamp' or $type == 'datetime' or $type == 'manjaro' or $type == 'recursive_ls' {
```

Comment:
Maybe just put `else` here? Seems reasonable anyway and a bit cleaner, and if a strange type is given, the healthcheck script will catch it anyway.